src/main/java/org/folio/inventory/common/WebContext.java

```diff
@@ -36,6 +37,11 @@ public String getUserId() {
     return getHeader(OKAPI_USER_ID_HEADER, "");
   }
 
+  @Override
+  public String getRequestId() {
+    return getHeader(OKAPI_REQUEST_ID);
```

Comment:
This is inconsistent. All other methods return "" if undefined. Can you change to

    return getHeader(OKAPI_REQUEST_ID, "");